src/store/model-store.js

```diff
@@ -49,6 +49,16 @@ class ModelStore extends Store {
     getModelByActionId(actionId) {
         return modelsByActionId[actionId];
     }
+
+    deleteModelByAddress(address){
+        let result = readAddress(address);
+        if(result.type !== 'model' ){
+            throw new Error('Address is not an model.');
+        }
+        delete models[result.index];
+        let itemsOfModel = ItemStore.getItems().filter(item => item.modelAddress === address);
+        itemsOfModel.forEach(item => ItemStore.deleteItemByAddress(item.address));
```

Comment:
~Noice~ anomaly mitigation
I'd say, "Why bother making an `itemsOfModel`, just murder the stuff out right".